src/language.js

```diff
@@ -1333,6 +1333,8 @@ const getValidationMessage = (validation, projectId = '') => {
     invalid_benthic_transect: () =>
       'One or more invalid fields: site, management, sample date, transect number, width, depth',
     invalid_depth: () => `Depth invalid or not greater than ${context?.depth_range[0]} m`,
+    excessive_precision: () =>
+      `Depth precision to right of decimal point greater than ${context?.decimal_places} digit(s)`,
```

Comment:
Does locale change number (decimals and commas) in the UI for other countries?  If so wording may need to change

Reply:
No atm we're not handling locale properly. There's a card to handle currencies with US-style commas/periods in the near term; hope is to come back to locales (and char sets, and translation) in a more thorough way next year. There are a lot of other things like this to consider.